functionnal_tests/cypress/integration/notification_wall/translation_spec.js

```diff
@@ -0,0 +1,21 @@
+import { PAGES } from '../../support/urls_commands'
+
+describe('Notification Wall', () => {
+  before(function () {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('users')
+    cy.visitPage({ pageName: PAGES.HOME })
+    this.skip() // FIXME - GB - 2020-08-10 - To be activate when the notification wall (https://github.com/tracim/tracim/issues/2840) and button's header (https://github.com/tracim/tracim/issues/3323) were merge
```

Comment:
```suggestion
    this.skip() // FIXME - GB - 2020-08-10 - To be enabled when the notification wall (https://github.com/tracim/tracim/issues/2840) and button's header (https://github.com/tracim/tracim/issues/3323) are merged
```
Maybe add a checkbox in these issues to enable these tests?

Reply:
The idea is enable this tests in this PR, I put the comment just to explain to the reviewers. But the skip it temporary, it will not be merge like this.

Reply:
ah, ok :-)
Let's use this conversation as a guard against a merge without this modification then.